lang/en.json

```diff
@@ -1060,6 +1060,12 @@
         "TagFireLimitAllowedInPreviewModeTitle": "Enable tag fire limits to work in preview mode",
         "TagFireLimitAllowedInPreviewModeDescription": "Enable this option to check if fire limits, such as once-per-lifetime, work as expected when preview mode is enabled.",
         "MatomoConfigurationMatomoTrackBotsTitle": "Track Bots",
-        "MatomoConfigurationMatomoTrackBotsDescription": "By default, Matomo doesn't track visits by bots in order to show accurate visitor metrics. Enable this feature to add tracking for bot visits."
+        "MatomoConfigurationMatomoTrackBotsDescription": "By default, Matomo doesn't track visits by bots in order to show accurate visitor metrics. Enable this feature to add tracking for bot visits.",
+        "ClickHtmlAttributeVariableName": "Clicked HTML Attribute",
+        "ClickHtmlAttributeVariableDescription": "Reads a custom value from the clicked HTML attribute.",
+        "ClickHtmlAttributeVariableHelp": "Using this variable you can access any value that is available within the clicked HTML attribute.",
+        "ClickHtmlAttributeAttributeTitle": "Clicked HTML Attribute Name",
+        "ClickHtmlAttributeAttributeDescription": "The name of the attribute, such as title, value, alt, or any other HTML attribute.",
+        "ClickHtmlAttributeAttributePlaceholder": "e.g. value"
```

Comment:
Just wanted to check is this a correct placeholder ?

Reply:
Yes. One of the tickets associated with this specifically wanted the ability to access the `value` attribute. I thought that it might be a good placeholder.